### ISSUE TITLE
ProductStickers

### ISSUE THREAD

**Issue by dionis154** [magento-hackathon/developer-paradise-2016#14]

**Comment by peterjaap** [comment]
You mean something like this (the 60 euro cashback 'sticker')? 

<img width="901" alt="screenshot 2016-04-25 10 21 25" src="https://cloud.githubusercontent.com/assets/431360/14777765/83a82704-0acf-11e6-9c48-f78199ffbe80.png">

http://www.mastertools.nl/bosch-cashback

?

**Comment by dionis154** [comment]
Yes. But the main idea is apply stickers for each products on events via php.
